### PR TITLE
adding models

### LINKED ISSUE #1: [FEAT] Add Models
Add models for 

* Client
* Order
* Product
* Category

Following the next image

image

### PR DESCRIPTION
**Pull request by Cletox**
first pull request

### CODE REVIEW

File: src/routes/db/productsModel.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const productsSchema = new Schema({
+  //_id: { type: mongoose.Types.ObjectId, required: true },
+  name: String,
+  model: String,
+  brand: String,
+  categoryId: String,
```
Comment (odarotto) [inline src/routes/db/productsModel.js]: `categoryId` should be `mongoose.Types.ObjectId` type

File: src/routes/db/productsModel.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  });
+
+
+const clientSchema  = new Schema({
+    //_id: { type: mongoose.Types.ObjectId, required: true },
+    name: String,
+    nid: String,
+    phone, String
```
Comment (odarotto) [inline src/routes/db/productsModel.js]: `phone, String` is a syntax error, this should be: `phone: String`

**Review by odarotto — COMMENTED** [review]
Please review changes requested

File: src/routes/db/productsModel.js (hunk 1 above)
Comment (Cletox) [reply src/routes/db/productsModel.js]: swapped.

File: src/routes/db/productsModel.js (hunk 2 above)
Comment (Cletox) [reply src/routes/db/productsModel.js]: done.

**Comment by odarotto** [comment]
lgtm

**Pull request merged**